Выполнение ТЗ 6 спринта.

SergikF opened the pull request:
Преобразование истории операций.
Изменение в тестах.
Изменение визуализации действия в main.

SergikF committed:
Выполнение ТЗ 6 спринта.
Преобразование истории операций.
Изменение в тестах.
Изменение визуализации действия в main.

SergikF committed:
Выполнение ТЗ 6 спринта.
Исправление для прохождения проверки GitHub

SergikF committed:
Выполнение ТЗ 6 спринта.
Исправление для прохождения проверки GitHub

SergikF committed:
Выполнение ТЗ 6 спринта.
Исправление для прохождения проверки GitHub

SergikF committed:
Выполнение ТЗ 6 спринта.
Исправление для прохождения проверки GitHub

SergikF committed:
Выполнение ТЗ 6 спринта.
Исправление для прохождения проверки GitHub

SergikF committed:
Выполнение ТЗ 6 спринта.
Исправление для прохождения проверки GitHub

SergikF committed:
Выполнение ТЗ 6 спринта.
Исправление ошибок стиля при прохождении проверки GitHub

johnskyi commented on src/main/service/TaskManagerImpl.java:
```diff
@@ -113,30 +120,32 @@ public Object getEpicSubTasks(int id) {
     @Override
     public boolean deleteTask(int id) {
         if (task.containsKey(id)) { // если id найдено - удаляем задачу
+            history.remove(id);
```
я бы посоветовал сместить метод удаления из истории чуть ниже, после удаления из хранилища, так как может возникнуть ошибка, а из истории уже удалится. И так во всех местах использования метода

SergikF replied:
Изменил очерёдность операции при удалении задач, подзадач и эпиков
- сначала удалить, а потом внести изменение в историю.

SergikF committed:
Выполнение ТЗ 6 спринта.
Исправление замечаний при проверке.
Изменил очерёдность операции при удалении задач, подзадач и эпиков
- сначала удалить, а потом внести изменение в историю.
